src/actions/ScanActions.js

```diff
@@ -93,9 +93,10 @@ export const doRequestAddress = async (dispatch: Dispatch, currencyWallets: { [w
     )) {
       const reqNativeCode = asset.nativeCode
       const reqTokenCode = asset.tokenCode
-      const pluginId = getPluginIdFromChainCode(reqNativeCode) ?? '' // Will never by empty string. Already filtered from for loop filter above
```

Comment:
why was the ?? removed. This can technically still return void

Reply:
We already check this with a call to `unsupportedNativeCodes` on line 80. I'll reorganize the logic here